app/src/main/java/org/mozilla/focus/browser/VideoVoiceCommandMediaSession.kt

```diff
@@ -118,6 +121,16 @@ class VideoVoiceCommandMediaSession @UiThread constructor(
         this.sessionIsLoadingObserver = null
     }
 
+    @OnLifecycleEvent(ON_RESUME)
```

Comment:
Maybe include a comment that onResume and onPause are related to interruptions of the voice commands, because I think it can be confusing since we also have onStart onStop, for actual play/pause events.

Reply:
This feels like overkill to me. The way I see it, if someone wants to know why these methods exist, they'd see what flag changes in it (`isResumed`) and see what uses that flag for why the flag needs to exist (and thus the methods).

Another concerns with adding a comment are:
- The boolean flag stands on its own and represents a state (`isResumed`) such that others can use it: the comment makes less sense if there are multiple consumers
- If a consumer is added and this consumer is removed, the comment may not be removed and then the comment is out-of-date. I'd rather comment directly on where the flag is used (as I have done).